### ISSUE TITLE
iris console spam and not working

### ISSUE THREAD

**Issue by ThebestkillerTBK** [kerudion/chunksfadein#22]
spams
```
[Render thread/INFO]: ----- VERTEX -----
[Render thread/INFO]: #version 330 core
``` 
and
 ```
[<sodium shaders>] Unsupported uniform: vec4 Chunk_FadeDatas[0].fadeData
```
in the console and chunksfadein wont work while shaders enabled

**Comment by kerudion** [comment]
Shader code is printed for debug purposes, "Unsupported uniform" comes from Iris because Chunk_FadeDatas uniform is not controlled by Iris code. I'll fix these messages in near future.

**Comment by DoTrONoL** [comment]
I only click to enable it and become gifted with 13k lines of log. 😂
https://mclo.gs/oh9y7S6

**Comment by DoTrONoL** [comment]
Yey, its fixed, no more console spam at 1.19.2. Also fixed my game's crash when I was trying to use the `VanillaPlus v3.0b`.

**Comment by ThebestkillerTBK** [comment]
yes. fixed in latest build.
